docs/content/direct/troubleshooting.md

```diff
@@ -0,0 +1,27 @@
+# Troubleshooting
+
+This guide is a work in progress.
+
+## Things to look at
+
+- Remember that for each of your BindingPolicy objects, there is a corresponding Binding object that reports what is matching the policy object.
+- Although not part of the interface, when debugging you can look at the ManifestWork and WorkStatus objects in the ITS.
+- Look at logs of controllers. If they have had container restarts that look relevant, look also at the previous logs. Do not forget OCM controllers.
+- If a controller's `-v` is not at least 4, increase it.
+- Remember that Kubernetes controllers tend to report transient problems as errors without making it clear that the problem is transient and tend to not make it clear if/when the problem has been resolved (sigh).
+
+## Making a good trouble report
+
+Basic configuration information.
+
+- Include the versions of all the relevant software; do not forget the OCM pieces.
```

Comment:
I think this, plus the user-friendly display of combinedstatuses, can kickoff the writing of KS cli tools.